app/controllers/v1/forams_controller.rb

```diff
@@ -3,7 +3,7 @@ class ForamsController < ApplicationController
     before_action :set_foram, only: :show
 
     def index
-      forams = ForamFilter.new(foram_filter_params).forams
+      forams = ForamFilter.new(foram_filter_params).forams(user: @current_user)
```

Comment:
Wydawało mi się, że jak wychodziłem z developmentu to jeszcze nie było tej metody; tak czy siak się zrebase'owałem teraz i naprawię